implementation/drone/components/radar/src/java/org/inaetics/dronessimulator/drone/components/radar/Activator.java

```diff
@@ -15,14 +16,19 @@ public void init(BundleContext bundleContext, DependencyManager dependencyManage
         dependencyManager.add(createComponent()
                 .setInterface(Radar.class.getName(), null)
                 .setImplementation(Radar.class)
+                .setCallbacks("init", "start", "stop", "destroy")
                 .add(createServiceDependency()
                         .setService(DroneInit.class)
                         .setRequired(true)
                 )
                 .add(createServiceDependency()
                         .setService(Subscriber.class)
                         .setRequired(true)
-                ).setCallbacks("init", "start", "stop", "destroy")
```

Comment:
Sure that the callbacks do not need to be set? I would think they are there for a reason.